ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-release-5.8.yaml

```diff
@@ -36,11 +36,11 @@ base_images:
     namespace: logging
     tag: logging-elasticsearch6
   logging-eventrouter:
-    name: 5.y
+    name: "5.8"
```

Comment:
eventrouter is built from master in our release, so there should be no separate `5.8` tag for this, using `5.y` should be fine.